src/data-loader.ts

```
@@ -4879,4 +4929,114 @@
+
+			// Save cost data first
+			await saveCostData('test-session-123', 15.75);
+
+			const cost = await calculateCostForEntry(mockUsageData, 'statusline', mockFetcher);
+			expect(cost).toBe(15.75); // Should use saved statusline cost
+		});
+
+		it('should fallback to costUSD in statusline mode when no saved cost', async () => {
+			const mockFetcher = new PricingFetcher(true);
+
+			const dataWithoutSavedCost = { ...mockUsageData, sessionId: 'non-existent-session' };
```

ESLint

[error] Unsafe assignment of an `any` value.

(ts/no-unsafe-assignment)